smoker/src/smoker/smoker/commands.go

```diff
@@ -157,6 +190,28 @@ func replDump(s []string, b backends.Backend) {
 	printDump(c)
 }
 
+func replSave(s []string, b backends.Backend) {
+	// TODO warn before overwriting
+	if len(s) < 1 {
```

Comment:
Not sure if I'm handling spaces correctly. Maybe I should merge the array together instead of taking the first elt.

Reply:
Since I'm using [strings.Fields](https://golang.org/pkg/strings/#Fields) to split, this method won't handle multiple whitespace correctly. Adding support for whitespace in path names will be too much work, moving to backlog.